std-bits/aws/src/main/java/org/enso/aws/ProfileReader.java

```diff
@@ -0,0 +1,20 @@
+package org.enso.aws;
+
+import software.amazon.awssdk.profiles.ProfileFileSupplier;
+
+public class ProfileReader {
+  public static final ProfileReader INSTANCE = new ProfileReader();
+
+  private final String[] profiles;
+
+  private ProfileReader() {
+
+    var pvdr = ProfileFileSupplier.defaultSupplier();
+    var profileFile = pvdr.get();
+    profiles = profileFile.profiles().keySet().toArray(new String[0]);
```

Comment:
I'm wondering what is the cost of reading these? Maybe we should load them lazily on first request instead of on load?